js/geo/lng_lat.js

```diff
@@ -18,6 +18,9 @@ var wrap = require('../util/util').wrap;
  * @param {number} lat Latitude, measured in degrees.
  * @example
  * var ll = new mapboxgl.LngLat(-73.9749, 40.7736);
+ * @see {@link https://www.mapbox.com/mapbox-gl-js/example/mouse-position/|Get coordinates of the mouse pointer}
```

Comment:
> Will jekyll `{{site.baseurl}}` work within the handlebars files like `section.hbs`? 
It should. 

I can see now how using `{{site.base}}` isn't immediately applicable in this use case. Relative URLs like you suggested are a good option.